create_deposition/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -ex
+curl --header "Content-Type: application/json" \
+      --request POST \
+      --data @$ZENODO_METADATA \
+      https://sandbox.zenodo.org/api/deposit/depositions?access_token=$ZENODO_API_TOKEN \
```

Comment:
I believe `sandbox` is only used for testing and should be removed when wanting to actually upload/create "real" archives. But since we need this for testing, we can define a hidden (undocumented) environment variable named `ZENODO_USE_SANDBOX` that, when given, would prepend `sandbox.` to the URL.

Thus, we need to update the workflows on `ci/` so they add this variable to their `env` attributes.